appinfo/info.xml

```diff
@@ -22,7 +22,7 @@
     <bugs>https://github.com/v1r0x/files_3d/issues</bugs>
     <screenshot>https://raw.githubusercontent.com/v1r0x/files_3d/master/screenshots/screenshot1.png</screenshot>
     <dependencies>
-        <nextcloud min-version="14" max-version="16"/>
+        <nextcloud min-version="16" max-version="19"/>
```

Comment:
Please use 17 btw as 16 is EOL in a few weeks ;)

Reply:
Actually the Viewer event might be for 18 only, so maybe your new viewer integration should come from 18 and above :)